test/base/test_diagitensor.jl

```diff
@@ -61,7 +61,7 @@ using Test
     @testset "diag" for ElType in (Float64, ComplexF64)
       A = diagITensor(randn(ElType, d), i, j)
       dA = diag(A)
-      @test dA isa DenseTensor{ElType,1}
+      @test dA isa NDTensors.DenseTensor{ElType,1}
```

Comment:
Is this no longer exported?

Reply:
DenseTensor is being exported but there is a weird issue with this line. I can also fix this issue by changing `using ITensors.NDTensors` to `using NDTensors`